frame/nomination-pools/src/lib.rs

```diff
@@ -1274,6 +1316,10 @@ pub mod pallet {
 			state_toggler: Option<T::AccountId>,
 			nominator: Option<T::AccountId>,
 		},
+		/// The active balance of pool `pool_id` has been slashed to `balance`.
+		PoolSlashed { pool_id: PoolId, balance: BalanceOf<T> },
```

Comment:
Maybe also include the current points?  
Then the current slash ratio is easily calculable by UI.